plugin/gatsby-node.js

```diff
@@ -123,6 +123,7 @@ const shopifyNodeTypes = [
   `ShopifyMetafield`,
   `ShopifyOrder`,
   `ShopifyProduct`,
+  `ShopifyProductImage`,
```

Comment:
@LekoArts I'm not sure what happened, I thought I'd added this already but git does not agree. Should fix the image problem you're having :crossed_fingers: